tessera-jaxrs/openapi/generate/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 configurations.all {
   resolutionStrategy {
-    force 'org.yaml:snakeyaml:1.31', 'com.fasterxml.jackson.core:jackson-databind:2.13.3'
+    force 'org.yaml:snakeyaml:1.33', 'com.fasterxml.jackson.core:jackson-databind:2.14.0-rc1'
```

Comment:
Are the rc versions explicitly required?

Reply:
2.14.0 hasn't been released yet https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind